firebase_admin/_sseclient.py

```diff
@@ -0,0 +1,198 @@
+"""SSEClient module to handle streaming of realtime changes on the database
+to the firebase-admin-sdk
+"""
+
+import re
+import time
+import warnings
+import six
+
+import requests
+
+
+# Technically, we should support streams that mix line endings.  This regex,
+# however, assumes that a system will provide consistent line endings.
+end_of_field = re.compile(r'\r\n\r\n|\r\r|\n\n')
+
+
+class KeepAuthSession(requests.Session):
+    """A session that does not drop Authentication on redirects between domains"""
+    def rebuild_auth(self, prepared_request, response):
+        pass
+
+
+class SSEClient(object):
+    """SSE Client Class"""
+    def __init__(self, url, session, build_headers, last_id=None, retry=3000, **kwargs):
+        self.should_connect = True
+        self.url = url
+        self.last_id = last_id
+        self.retry = retry
+        self.running = True
+        # Optional support for passing in a requests.Session()
+        self.session = session
+        # function for building auth header when token expires
+        self.build_headers = build_headers
```

Comment:
Do we really need this? The session we get from the db module is an instance of `google.auth.transport.requests.AuthorizedSession`, which is guaranteed to add the required Authorization header. So this shouldn't be required.

Reply:
`build_header()` and all subsequent function arguments removed.